plugins/modules/aci_fabric_span_src_group.py

```diff
@@ -0,0 +1,300 @@
+  type: int
+  sample: 200
+url:
+  description: The HTTP url used for the request to the APIC
+  returned: failure or debug
+  type: str
+  sample: https://10.11.12.13/api/mo/uni/tn-production.json
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible_collections.cisco.aci.plugins.module_utils.aci import ACIModule, aci_argument_spec, aci_annotation_spec, aci_owner_spec
+
+
+def main():
+    argument_spec = aci_argument_spec()
+    argument_spec.update(aci_annotation_spec())
+    argument_spec.update(aci_owner_spec())
+    argument_spec.update(
+        source_group=dict(type="str", aliases=["name", "src_group"]),  # Not required for querying all objects
+        description=dict(type="str", aliases=["descr"]),
+        admin_state=dict(type="bool"),
+        destination_group=dict(type="str", aliases=["dst_group"]),
+        state=dict(type="str", default="present", choices=["absent", "present", "query"]),
+        name_alias=dict(type="str"),
+    )
+
+    module = AnsibleModule(
+        argument_spec=argument_spec,
+        supports_check_mode=True,
+        required_if=[
+            ["state", "absent", ["source_group"]],
+            ["state", "present", ["source_group", "destination_group"]],
+        ],
+    )
+
+    aci = ACIModule(module)
+
+    source_group = module.params.get("source_group")
+    description = module.params.get("description")
+    admin_state = aci.boolean(module.params.get("admin_state"), "enabled", "disabled")
+    destination_group = module.params.get("destination_group")
+    state = module.params.get("state")
+    name_alias = module.params.get("name_alias")
+
+    aci.construct_url(
+        root_class=dict(
+            aci_class="fabric",
+            aci_rn="fabric",
+        ),
+        subclass_1=dict(
+            aci_class="spanSrcGrp",
+            aci_rn="srcgrp-{0}".format(source_group),
+            module_object=source_group,
+            target_filter={"name": source_group},
+        ),
+        child_classes=["spanSpanLbl"],
+    )
+
+    aci.get_existing()
+
+    if state == "present":
+        # Create new child configs payload
+        child_configs = [{"spanSpanLbl": {"attributes": {"name": destination_group}}}]
+
+        # Validate if existing and remove child object when is does not match provided configuration
+        if isinstance(aci.existing, list) and len(aci.existing) > 0:
+            for child in aci.existing[0].get("spanSrcGrp", {}).get("children", {}):
+                if child.get("spanSpanLbl") and child.get("spanSpanLbl").get("attributes").get("name") != destination_group:
+                    child_configs.append(
+                        {
+                            "spanSpanLbl": {
+                                "attributes": {
+                                    "dn": "uni/fabric/srcgrp-{0}/spanlbl-{1}".format(source_group, child.get("spanSpanLbl").get("attributes").get("name")),
+                                    "status": "deleted",
```

Comment:
Does status: "deleted" work for this child object unlike the child in the aci_access_span?

Reply:
This is done the same as in access span. This class, spanSpanLbl, which has a name identifier is allowed to be deleted this way.